colab/nessie-iceberg-flink-demo-nba.ipynb

```diff
@@ -0,0 +1,565 @@
+{
```

Comment:
I assume, this one does the same thing as the initial notebook, just with Flink?

Reply:
more or less, yes.